aardvark/updater/__init__.py

```diff
@@ -44,15 +51,15 @@ def update_account(self):
         arns = self._get_arns()
 
         if not arns:
-            self.current_app.logger.warn("Zero ARNs collected. Exiting")
+            log.warning("Zero ARNs collected. Exiting")
             exit(-1)
```

Comment:
Mixing of returns, exits, and exceptions. Might want to consider consistent way to handle failures

Reply:
Yeah, I don't like this. I left most of the updater logic as it was since that's slated for refactoring next.

Reply:
To be addressed in #116